### PR TITLE
[assignee_no_login] Consider all types of inactivity

### LINKED ISSUE #1379: Take into account all kinds of inactivity for assignee_no_login
Right now it's only taking into account people who didn't login for a long time, it could instead use all kinds of inactivity just like the triage owner script.

### PR DESCRIPTION
**Pull request by suhaibmujahid**
Resolves #1379 

## Autonag wiki page
```diff
{{AutonagRule
  | Rule name = Assignee is inactive
  | Purpose   = Help identify bugs that are assigned to inactive accounts
- | Action    = Needinfo the triage owner
+ | Action    = Reset the assignee, and if the bug has high priority or high severity, needinfo the triage owner
  | Source    = assignee_no_login.py
}}
```

## Dry-run

```
2022-06-09 18:13:18,677 - INFO - Run tool assignee_no_login.py
2022-06-09 18:14:06,704 - INFO - Tool assignee_no_login.py has finished.
```
[dryrun.html.txt](https://github.com/mozilla/relman-auto-nag/files/8874424/dryrun.html.txt)

## Checklist

- [x] Type annotations added to new functions
- [x] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results

### CODE REVIEW

File: auto_nag/scripts/assignee_no_login.py [hunk 1]
```diff
@@ -110,9 +153,6 @@ def get_bz_params(self, date):
         params = {
             "include_fields": fields,
             "resolution": "---",
-            "f1": "assignee_last_login",
-            "o1": "lessthan",
-            "v1": start_date,
```
Comment (marco-c) [inline auto_nag/scripts/assignee_no_login.py:115]: How long is the query taking now? It is super broad, maybe we should exclude recently touched bugs?

Comment (suhaibmujahid) [reply auto_nag/scripts/assignee_no_login.py:115]: > How long is the query taking now? 

It does not take much, based on the dryrun above, it took 48 seconds. 

> It is super broad, maybe we should exclude recently touched bugs?

We already exclude bugs that got assigned recently (not general activities):

https://github.com/mozilla/relman-auto-nag/blob/5c7aab92e94d14f61e46f80cebd96dcedca92fdf/auto_nag/scripts/assignee_no_login.py#L157-L159

Comment (marco-c) [reply auto_nag/scripts/assignee_no_login.py:115]: All right, 48 seconds is acceptable.

**Review by marco-c — APPROVED** [review]
LGTM!

**Pull request merged**